src/routing_utils/nats_client/main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+
+	"code.cloudfoundry.org/tlsconfig"
+	"github.com/nats-io/nats.go"
+	"gopkg.in/yaml.v2"
+)
+
+const USAGE = `Usage:
+/var/vcap/jobs/gorouter/bin/nats_client [COMMAND] [SUBJECT] [MESSAGE]
+
+COMMANDS:
+  subscribe    (Default) Streams NATS messages from server with provided SUBJECT. Default SUBJECT is 'router.*'
+               Example: /var/vcap/jobs/gorouter/bin/nats_client subscribe 'router.*'
+
+  publish      Publish the provided message JSON to SUBJECT subscription. SUBJECT and MESSAGE are required
+               Example: /var/vcap/jobs/gorouter/bin/nats_client publish router.register '{"host":"172.217.6.68","port":80,"uris":["bar.example.com"]}'
+`
+
+// Simple NATS client for debugging
+// Uses gorouter.yml for config
+func main() {
+	if os.Args[len(os.Args)-1] == "--help" || os.Args[len(os.Args)-1] == "-h" || os.Args[len(os.Args)-1] == "help" {
```

Comment:
Because I wanted to keep the number of dependencies small and keep the implementation as simple as possible.

I'm happy to add a args library if you prefer though

Reply:
We can leave it like it is for now and consider adding a flag parser once we see the need for additional arguments.